src/meta/proto-conv/tests/it/user_proto_conv.rs

```diff
@@ -160,6 +161,42 @@ pub(crate) fn test_gcs_stage_info() -> mt::UserStageInfo {
     }
 }
 
+// Version 13: add OSS as a stage backend
+pub(crate) fn test_oss_stage_info() -> mt::UserStageInfo {
```

Comment:
If this is version specific, add the version to the method name instead of to comment.